pipeline-scripts/release.groovy

```diff
@@ -116,6 +116,22 @@ Map stageValidation(String quay_url, String dest_release_tag, int advisory = 0,
         error("🚫 Advisory ${advisoryInfo.id} doesn't seem to be associated with a live ID.")
     }
 
+    if (arch == 'amd64' || arch == 'x86_64') {
+        echo "Verifying payload"
+        res = commonlib.shell(
+            returnAll: true,
+            script: "elliott --group=openshift-${version} verify-payload registry.svc.ci.openshift.org/ocp/release:${nightly} ${advisory}"
```

Comment:
now each arch has it's own directory path like
registry.svc.ci.openshift.org/ocp-s390x/release-s390x:4.5.0-0.nightly-s390x-2020-09-14-030917
registry.svc.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.5.0-0.nightly-ppc64le-2020-09-14-124036
registry.svc.ci.openshift.org/ocp/release:4.5.0-0.nightly-2020-09-14-124053

Reply:
true but as noted above this only currently works correctly with x86.